examples/base/units.py

```diff
@@ -62,20 +65,42 @@
 
 # %%[markdown]
 # ## Parameters and Units
-# First I grab the default configuration
+# First I make a small ParameterFrame
 
 # %%
 
-pf = cfg.Configuration()
+
+@dataclass
+class MyParameterFrame(ParameterFrame):
+    """A ParameterFrame"""
+
+    A: Parameter[float]
+
+
+# this works the same
+@parameter_frame
+class MyDecoratedFrame:
+    """A ParameterFrame made with a decorator"""
+
+    A: Parameter[float]
+
+
+mypf = MyParameterFrame.from_dict({"A": {"value": 5, "unit": ""}})
+mydecpf = MyDecoratedFrame.from_dict({"A": {"value": 5, "unit": ""}})
+
+print(mypf)
+print(mydecpf)
+# Both frames equal
+assert all([dparam == param for dparam, param in zip(mydecpf, mypf)])  # noqa: S101
```

Comment:
I'm fairly sure you can do `mypf == mydecpf` here

Reply:
![image](https://user-images.githubusercontent.com/81617086/196461734-cbed0088-1e16-4a5a-91f9-9f76ed154f48.png)

Apparently not...

Reply:
Oh dear, that's not good

Reply:
I've made an issue for now see #1539